README.rst

```diff
@@ -75,9 +75,46 @@ From System Package Manager
 ---------------------------
 
 If your operating system packages repolib, you can install it by running::
-    
+
     sudo apt install python3-repolib
 
+If your distro does not yet supply the package, you can install the
+[latest release](https://github.com/pop-os/repolib/releases/) using the
+steps below (At the time of writing this, the version is 2.0.0):
```

Comment:
Change "steps below" to "script below" since you're using `apt-get` instead of `apt` as if this is a machine script rather than individual commands for a user to run.